tests/lib/tools/setup_nested_hybrid_system.sh

```diff
@@ -20,6 +20,9 @@ run_muinstaller() {
     local label="${7}"
     local disk="${8}"
     local kern_mods_comp="${9:-}"
```

Comment:
otherwise the parameter indices and the shifting, combined with set -u make no sense. 9th parameter cannot be optional and have a default value expansion if 10th is expected to be present and if it's missing the code fails due to errors on unset variables
```suggestion
    local kern_mods_comp="${9}"
```